yolox/exp/base_exp.py

```diff
@@ -20,7 +20,7 @@ class BaseExp(metaclass=ABCMeta):
 
     def __init__(self):
         self.seed = None
-        self.output_dir = "/data/YOLOX_outputs"
+        self.output_dir = "./data/YOLOX_outputs"
```

Comment:
Hello, could you please change to
```suggestion
        self.output_dir = "./YOLOX_outputs"
```